includes/admin/importers/class-wc-product-wp-importer.php

```diff
@@ -15,7 +15,7 @@
  * @package     WooCommerce/Admin/Importers
  * @version     3.1.0
  */
-class WC_Product_Importer extends WP_Importer {
+class WC_Product_WP_Importer extends WP_Importer {
```

Comment:
We can split after, still I needed the class name for the abstarct class.
And this class already comes from previous Pull Requests.